src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 7th point. You should also connect `Avenir-Heavy` for bolder font-weight, but it should have te same font-family. So as the result you would have 2 `@font-face` declarations